cluster-provision/okd/scripts/provision.sh

```diff
@@ -122,6 +122,10 @@ envsubst < /manifests/okd/registries.yaml > /registries.yaml
 set +x
 export PULL_SECRET=$(cat /etc/installer/token)
 export SSH_PUBLIC_KEY="ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEA6NF8iallvQVp22WDkTkyrtvp9eWW6A8YVr+kz4TjGYe7gHzIw+niNltGEFHzD8+v1I2YJ6oXevct1YeS0o9HZyN1Q9qgCgzUFtdOKLv6IedplqoPkcmF0aYet2PkEDo3MlTBckFXPITAMzF8dJSIFo9D8HfdOV0IAdx4O7PtixWKn5y2hMNG0zQPyUecp4pzC6kivAIhyfHilFR61RGL+GPXQ2MWZWFYbAGjyiYJnAmCP3NOTd0jMZEnDkbUvxhMmBYSdETk1rRgm+R4LOzFUGaHqHDLKLX+FIPKcF96hrucXzcWyLbIbEgE98OHlnVYCzRdK8jlqm8tehUc9c9WhQ== vagrant insecure public key"
+
+# For ocp-4.4 we want OVNKubernetes
```

Comment:
I don't see how is this comment relevant or connected to the next line. How would someone use OVN networking? Do we need another provider for that?

Reply:
This is old stuff, let's me move it to the proper place.

Reply:
Ohh gosh I have "export NETWORKING_TYPE=OpenShiftSDN" harcoded at provision.sh